src/libprojectM/Renderer/ShaderEngine.cpp

```diff
@@ -19,8 +19,10 @@
 
 #ifdef USE_GLES
     #define GLSL_VERSION "300 es"
+    #define SHADER_VERSION  M4::GLSLGenerator::Version_300_ES
```

Comment:
i think this still won't fly on a raspi will it?

Reply:
no, hopefully gles3 support will be implemented on broadcom vc4 driver someday.